### PR TITLE
Support visualization and dynamic in AMC

### PR DESCRIPTION
**Pull request by pakrym**
BinaryData is solution for all problems

### CODE REVIEW

File: sdk/monitor/Azure.Monitor.Query/src/Models/LogsQueryResults.cs [hunk 1]
```diff
@@ -24,5 +27,10 @@ public partial class LogsQueryResult
         /// Returns the query statistics if the <see cref="LogsQueryOptions.IncludeStatistics"/> is set to <c>true</c>.
         /// </summary>
         public BinaryData Statistics => _statistics.ValueKind == JsonValueKind.Undefined ? null : new BinaryData(_statistics.ToString());
+
+        /// <summary>
+        /// Returns the query visualization if the <see cref="LogsQueryOptions.IncludeVisualization"/> is set to <c>true</c>.
```
Comment (christothes) [inline sdk/monitor/Azure.Monitor.Query/src/Models/LogsQueryResults.cs:32]: What does it return if IncludeVisualization is false?

File: sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientClientLiveTests.cs [hunk 2]
```diff
@@ -479,6 +489,29 @@ public async Task CanQueryWithStatistics(bool include)
...
+            });
+
+            if (include)
+            {
+                using JsonDocument document = JsonDocument.Parse(response.Value.Visualization);
+                Assert.AreNotEqual(JsonValueKind.Undefined, document.RootElement.GetProperty("visualization").ValueKind);
+            }
+            else
+            {
+                Assert.AreEqual(default, response.Value.Statistics);
```
Comment (christothes) [inline sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientClientLiveTests.cs:511]: Statistics?

File: sdk/monitor/Azure.Monitor.Query/src/Models/LogsQueryResults.cs (hunk 1 above)
Comment (pakrym) [reply sdk/monitor/Azure.Monitor.Query/src/Models/LogsQueryResults.cs:32]: Null. I think what you are trying to say is that I should add that to XML doc. 🤔

Comment (christothes) [reply sdk/monitor/Azure.Monitor.Query/src/Models/LogsQueryResults.cs:32]: > Null. I think what you are trying to say is that I should add that to XML doc. 🤔

**Review by christothes — APPROVED** [review]

**Pull request merged**